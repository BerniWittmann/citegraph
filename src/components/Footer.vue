<template>
  <v-footer
    app
    inset
  >
    <v-spacer></v-spacer>
    <div>{{ $t('footer.version', { version: version }) }}</div>
  </v-footer>
</template>

<style scoped lang="scss">
</style>
<script lang="ts">
import packageInfo from '@/../package.json'

import { Component, Vue } from 'vue-property-decorator'

@Component
export default class Footer extends Vue {
  get version (): string {
    return packageInfo.version
  }
}
</script>
