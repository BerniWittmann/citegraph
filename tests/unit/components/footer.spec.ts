import { shallowMount, RouterLinkStub } from '@vue/test-utils'
import { i18n } from '../setupPlugins'

import Footer from '@/components/Footer.vue'

describe('components/Footer.vue', () => {
  it('renders', () => {
    const wrapper = shallowMount(Footer, {
      i18n,
      stubs: {
        RouterLink: RouterLinkStub
      }
    })
    expect(wrapper.html()).toMatchSnapshot()
  })
})
