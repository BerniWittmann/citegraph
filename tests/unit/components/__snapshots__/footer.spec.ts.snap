// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Footer.vue renders 1`] = `
<v-footer-stub height="auto" tag="div" tile="true" app="true" inset="true">
  <v-spacer-stub></v-spacer-stub>
  <div>footer.version</div>
</v-footer-stub>
`;
